app/assets/stylesheets/shared/_surveys.scss

```diff
@@ -0,0 +1,74 @@
+.survey {
+  $base-border-radius: 3px;
+  $base-background-color: #ffffff;
+  $dark-gray: #333;
+  $light-gray: #ddd;
+  $base-font-color: $dark-gray;
+
+  a {
+    text-decoration: none;
+  }
+
+  $survey-box-border-color: $light-gray;
+  $survey-box-border: 1px solid $survey-box-border-color;
+  $survey-box-background: lighten($light-gray, 10%);
+  $survey-box-gutter: 0.4em 0.4em 1.3em;
+  $survey-box-width: 18em;
+  $survey-box-color: transparentize($base-font-color, 0.3);
+
+  @include transition (all 0.2s ease-in-out);
+  background: $survey-box-background;
+  border-radius: $base-border-radius;
+  box-shadow: inset 0 0 1px $survey-box-border-color, 0 2px 4px darken($base-background-color, 15%);
+  display: block;
+  margin: $survey-box-gutter;
+  padding: 2em 2em 3em 2em;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `2em 2em 3em` instead of `2em 2em 3em 2em`